wagtail/admin/templates/wagtailadmin/shared/breadcrumb-next.html

```diff
@@ -0,0 +1,58 @@
+{% load i18n wagtailadmin_tags %}
+
+{% with breadcrumb_link_classes='w-flex w-justify-start w-items-center w-no-underline hover:w-underline w-text-grey-600 hover:w-text-primary' breadcrumb_list_classes='w-hidden w-overflow-hidden w-max-w-0 w-transition-all w-duration-300 w-whitespace-nowrap w-flex-shrink-0' icon_classes='w-w-4 w-h-4 w-mr-3' %}
+    {# Breadcrumbs are visible on mobile by default but hidden on desktop #}
+    <button data-toggle-breadcrumbs  {# handled in page-editor.js #}
+        class="w-box-border w-ml-0 w-p-4 w-h-full w-bg-transparent w-text-grey-400 w-transition hover:w-scale-110 hover:w-text-primary w-outline-offset-inside"
+        aria-label="{% trans 'Toggle breadcrumbs' %}"
+        aria-expanded="false"
+    >
+        {% icon name="dots-horizontal" class_name="w-w-4 w-h-4" %}
+    </button>
+
+    <div class="w-relative w-h-full w-bg-grey-50 w-top-0 w-z-20 w-flex w-items-center w-flex-row w-overflow-x-auto w-transition w-duration-300">
+        <nav data-breadcrumbs
+            class="breadcrumb-next-nav w-flex w-items-center w-flex-row w-overflow-x-auto"
+            aria-label="{% trans 'Breadcrumb' %}">
+            <ul class="w-flex w-flex-row w-justify-start w-items-center w-pl-0 w-gap-2 sm:w-gap-0 sm:w-space-x-2 w-mr-4">
+                {% for page in pages %}
+                    {% if page.is_root %}
+                        {# Root section is shown as a 'site' icon in place of the title #}
+                        <li class="{{ breadcrumb_list_classes }}" data-breadcrumb-link>
+                            <a class="{{ breadcrumb_link_classes }}" href="{% url 'wagtailadmin_explore_root' %}" tabindex="0">
```

Comment:
```suggestion
                            <a class="{{ breadcrumb_link_classes }}" href="{% url 'wagtailadmin_explore_root' %}" 
                            >
```

Reply:
Oops thanks for catching this. Sorry for missing that, I did mean to remove it 😇